lib/features/launches/presentation/screen/launches_screen.dart

```diff
@@ -1,27 +1,52 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:mentorship_e1_g3/core/di/dependency_injection.dart';
+import 'package:mentorship_e1_g3/core/widgets/custom_error_widget.dart';
+import 'package:mentorship_e1_g3/core/widgets/custom_loading_widget.dart';
 import 'package:mentorship_e1_g3/core/widgets/launch_item.dart';
+import 'package:mentorship_e1_g3/features/home/logic/cubit/home_cubit.dart';
+import 'package:mentorship_e1_g3/features/home/logic/cubit/home_state.dart';
 
-import '../widgets/custom_launch_app_bar.dart';
-
-class LaunchesScreen extends StatelessWidget {
+class LaunchesScreen extends StatefulWidget {
   const LaunchesScreen({super.key});
 
   @override
-  Widget build(BuildContext context) {
-    return Column(
-      children: [
-        const CustomLaunchAppBar(),
-        Expanded(
-          child: ListView.builder(
-            physics: const BouncingScrollPhysics(),
-            itemBuilder: (BuildContext context, int index)=>const Padding(
-              padding:  EdgeInsets.all(12.0),
-              child: LaunchItem(),
-            ),
-            itemCount: 10,
-          ),
-        ),
-      ],
-    );
+  State<LaunchesScreen> createState() => _LaunchesScreenState();
+}
+
+class _LaunchesScreenState extends State<LaunchesScreen> {
+  @override
+  void initState() {
+    super.initState();
+    getIt.get<HomeCubit>().emitGetLaunches();
```

Comment:
why this line ?

u can make this : 

context.read<HomeCubit>().emitGetLaunches();